### DISCUSSION TITLE
How to make the main window to dark color?

### DISCUSSION THREAD

**Discussion by elkan1788** [doublecmd/doublecmd#1423] [General]
Hi guys:

There is my first time to used the DC programe and I had found that the offical snapshot image show that the DC could be dark mode like below style:
![MainWindowDark.png](https://doublecmd.sourceforge.io/gallery/images/MainWindowDark.png)

But I tried all settings which work for color, but stilled not same result as above snapshot image, and only change the panel color and text color, Is there someone can help make it true? 
![image](https://github.com/doublecmd/doublecmd/assets/1217096/7770356a-bd81-4090-8205-654c55afba47)

Thanks very much.

**Comment by alexx2000** [comment]
Hi,

![123](https://github.com/doublecmd/doublecmd/assets/13401398/ee3ca946-6835-4072-959f-bf82b60dd7b9)

https://sourceforge.net/p/doublecmd/wiki/Download-Beta/
